manifests/m/Microsoft/PowerShell/7.3.6.0/Microsoft.PowerShell.installer.yaml

```diff
@@ -0,0 +1,30 @@
+# Created with Komac v1.9.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.PowerShell
+PackageVersion: 7.3.6.0
+Platform:
+- Windows.Universal
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- pwsh
+PackageFamilyName: Microsoft.PowerShell_8wekyb3d8bbwe

```

Comment:
```suggestion
```
Not needed now to avoid messing up package correlation, can be added later when msix are released